--- python_files/psutil/_pslinux.py.orig	2018-05-23 02:56:42.366195862 +0000
+++ python_files/psutil/_pslinux.py	2018-05-23 02:56:58.489424487 +0000
@@ -23,8 +23,14 @@
 
 from . import _common
 from . import _psposix
-from . import _psutil_linux as cext
-from . import _psutil_posix as cext_posix
+try:
+    from . import _psutil_linux as cext
+except ImportError:
+    import _psutil_linux as cext
+try:
+    from . import _psutil_posix as cext_posix
+except ImportError:
+    import _psutil_posix as cext_posix
 from ._common import ENCODING
 from ._common import ENCODING_ERRS
 from ._common import isfile_strict