--- _testcapimodule.c-orig	2018-10-05 14:24:52.151333000 +0000
+++ _testcapimodule.c	2018-10-05 14:25:55.281320000 +0000
@@ -10,7 +10,17 @@
 #include "Python.h"
 #include <float.h>
 #include "structmember.h"
+/* Differentiate between building the core module and building extension
+ * modules.
+ * https://bugs.python.org/issue19348
+ */
+#ifdef Py_BUILD_CORE
+#undef Py_BUILD_CORE
 #include "datetime.h"
+#define Py_BUILD_CORE
+#else
+#include "datetime.h"
+#endif
 #include "marshal.h"
 #include <signal.h>
 